lib/std/private/digitsutils.nim

```diff
@@ -46,7 +46,7 @@ func addChars[T](result: var string, x: T, start: int, n: int) {.inline.} =
     for i in 0..<n: result[old + i] = x[start + i]
   when nimvm: impl
   else:
-    when defined(js) or defined(nimscript): impl
+    when not declared(copyMem): impl
```

Comment:
I personally prefer flipping clauses to remove negations, I understand that this keeps the change smaller.